composer/callbacks/checkpoint_saver.py

```diff
@@ -267,9 +272,9 @@ class CheckpointSaver(Callback):  # noqa: D101
             remote file systems.
 
     Attributes:
-        saved_checkpoints (List[Tuple[Timestamp, List[pathlib.Path]]]): The checkpoint timestamps and filepaths.
+        saved_checkpoints (List[str]): The checkpoint filepaths.
```

Comment:
were these timestamps used anywhere? this is an api change, so we might need to not do it?

Reply:
I don't think they are. I guess I was trying to get pyright to work

Reply:
I think we should probably put it back in case someone has code relying on the timestamps being present in this attribute

Reply:
It was actually removed [here](https://github.com/mosaicml/composer/pull/1428/files#diff-b3be1fd7d6bb329e7dfcb406b7996e3ef170416faf5524caf50a451333b5f29eL314-R316) and the docstring is just an artifact

Reply:
but I reverted it anyway

Reply:
Oh wait, we should make the doctoring correct. Sorry I thought you changed the implementation. the perils of reviewing on my phone